app/views/admin/versions/index.html.haml

```diff
@@ -0,0 +1,48 @@
+.row
+  .col-md-12
+    .page-header
+      %h1 Revision History
+      %p.text-muted
+        Log of changes made to conferences and associated resources
+
+- @versions.each do |version|
+  .row
+    .col-md-9
+      %p
+        = version.whodunnit.nil? ? 'Someone (probably via the console)' : User.find(version.whodunnit).name
+
+        - if version.item_type == 'Event'
+          = event_change_description(version)
+
+        - else
+          - if version.event == 'create'
+            = 'created new'
+          - elsif version.event == 'update'
+            = "updated #{updated_attributes(version)} of"
+          - else
+            = 'deleted'
+
+        =  render partial: 'object_desc_and_link', locals: { version: version }
+
+        %span.text-muted
+          = distance_of_time_in_words(Time.now,version.created_at) + ' ago'
+
+    .col-md-3
+      .btn-group{role: "group"}
+        %a.btn.btn-success.btn-sm.show-changeset{id: "#{version.id}"} View Changes
+        %button.btn.btn-default.dropdown-toggle.btn-sm.btn-primary{"data-toggle" => "dropdown", :type => "button"}
```

Comment:
single quotes, and new ruby syntax style.
You can have
`'type' => 'button' `or `type: 'button'`